### DISCUSSION TITLE
#45 - Magical Tree

### DISCUSSION THREAD

**Discussion by rafehimself** [cssbattle/forum#140] [Bugs reports]
`<style>html{margin:0 155 180;color:1A4341;background:#1a4341;box-shadow:0 0 0 30px#998235,0 0 0 60px,0 0 0 90px#f3ac3c,0 0 0 120px,0 0 0 150px#998235;clip-path:inset(-2in -90px)}*>*{margin:0 30 -190;background:#f3ac3c`

I found that solution on github, shorter than my original,
but gives only 168.73 (67.5% match) despite being 100% match,
doesn't see the clip-path? would be my guess
![image](https://user-images.githubusercontent.com/58576132/137652926-0c12d5fa-9753-4ea5-a64a-d07bcb67bae8.png)

**Comment by gunnarbittersmann** [comment]
Maybe this solution had worked with the old backend a couple of months ago. For why it doesn’t anymore, check @alexzaworski’s [previewer](https://codepen.io/alexzaworski/full/vwmNKr).
